examples/to-do/index.es

```diff
@@ -0,0 +1,78 @@
+Element `to-do`
+
+(class extends Storable (HTMLElement) {
+
+  initialize
+    (tasks = { tasks: [{ task: 'Eat Food' }] })
+      { this.retrieve (tasks) }
+
+
+  onsubmit (event, input = this.select `input`) {
+    event.preventDefault () // ...form from submitting
+
+    this.context.tasks.push ({ task: input.value })
+
+    this.store () // save context
+    input.value = ''
+  }
+
+
+  onidle (mark) {
```

Comment:
@brandondees it runs after render. We have to do this or creates logic in view. Currently this operation takes about 100mu